### PR TITLE
current resprites using retro style

### PR DESCRIPTION
**Pull request by Crysoskis**
:cl: Crysoskis 
imageadd: New sprites for all PDAs
imageadd: New sprite for the PDA painter
imageadd: New sprite for the autolathe
imageadd: New sprite for the turret control panel
imageadd: New sprites for most of telecommunications
/:cl:

[why]: # current sprites for the new retro style ready to be implemented. if style is agreed upon more will come, and revisions will be made to previous sprites if need be.

### CODE REVIEW

**Comment by Battletrap360** [comment]
NICE!

**Review by alexkar598 — CHANGES_REQUESTED** [review]
Only include sprites your changing in the sunset dmi files

File: sunset/code/game/machinery/_machinery.dm [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/obj/machinery
+	icon = 'sunset/icons/obj/stationobjs.dmi'
```
Comment (alexkar598) [inline sunset/code/game/machinery/_machinery.dm]: you really shouldnt be overriding /obj/machinery

Comment (alexkar598) [inline sunset/code/game/machinery/_machinery.dm]: dont override /obj/machinery

**Review by alexkar598 — CHANGES_REQUESTED** [review]

**Review by alexkar598 — APPROVED** [review]

**Review by alexkar598 — CHANGES_REQUESTED** [review]
you need to fix tcomms icons,you are overriding the whole thing not individually

**Review by alexkar598 — APPROVED** [review]

**Comment by alexkar598** [comment]
travis is mean boi

**Comment by ghost** [comment]
gonna let icondiffbot do it's thing

**Comment by alexkar598** [comment]
icondiffbot went on vacation

**Comment by ghost** [comment]
Testmerged.

**Comment by ghost** [comment]
FINALLY, ICONDIFFBOT DID IT'S SHIT

**Review by ghost — APPROVED** [review]
Sprites are well received. Merging.

**Pull request merged**